### PR TITLE
reference/tools/tidb-binlog: add config items for Reparo

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Add config items for Reparo.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1935

### Which version does your change affect? 

dev, v2.1, v3.0

After the dev updates get 2 LGTMs, I'll update the v2.1 and v3.0 repos.

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 1]
```diff
@@ -46,6 +48,8 @@ Usage of Reparo:
     If it is not set, the recovery process ends up with the last binlog file.
 -safe-mode bool
     Specifies whether to enable safe mode. When enabled, it supports repeated replication.
+-txn-batch int
+    The number of SQL statements of a transaction that is output to the downstream database (20 by default).
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    The number of SQL statements in a transaction that is output to the downstream database (`20` by default).
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 2]
```diff
@@ -83,6 +87,12 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# The number of SQL statements of a transaction that is output to the downstream database (20 by default).
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# The number of SQL statements in a transaction that is output to the downstream database (`20` by default).
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 3]
```diff
@@ -83,6 +87,12 @@ log-level = "info"
...
+txn-batch = 20
+
+# The number of concurrencies of the downstream for the replication process (16 by default). A higher value indicates a better throughput for the replication (16 by default).
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# The number of concurrencies in the downstream for the replication process (`16` by default). A higher value indicates a better throughput for the replication.
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 4]
```diff
@@ -20,6 +20,8 @@ Usage of Reparo:
     The level of the output information of logs
     Value: "debug"/"info"/"warn"/"error"/"fatal" ("info" by default)
 -V Prints the version.
+-c int
+    The number of concurrencies of the downstream for the replication process (16 by default). A higher value indicates a better throughput for the replication.
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    The number of concurrencies in the downstream for the replication process (`16` by default). A higher value indicates a better throughput for the replication.
```

**Comment by CaitinChen** [comment]
@yikeke Thanks. PTAL again.

**Pull request merged**